visualisation/visualise-raw-data/visualise-raw-data-functions.R

```diff
@@ -0,0 +1,72 @@
+plot_raw_data = function(national = TRUE,
+                         cutoff = 25){
+  
+  # Get observed data ------------------------------------------------------------------
+  source(here::here("utils", "get-us-data.R"))
```

Comment:
not super keen about sourcing a function in a function. Perhaps do it at the top of the script